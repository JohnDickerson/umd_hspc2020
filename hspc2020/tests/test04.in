10
David Wallace
Louanne Kelley
Jo Bennett
Cathy Simms
Devon White
Danny Cordray
Karen Filippelli
Rolando
Stephanie
Ben Nugent
