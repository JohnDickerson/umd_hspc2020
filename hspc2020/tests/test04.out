Name David Wallace can generate 6652800 unique possibilities.
Name Louanne Kelley can generate 39916800 unique possibilities.
Name Jo Bennett can generate 10080 unique possibilities.
Name Cathy Simms can generate 362880 unique possibilities.
Name Devon White can generate 362880 unique possibilities.
Name Danny Cordray can generate 4989600 unique possibilities.
Name Karen Filippelli can generate 1210809600 unique possibilities.
Name Rolando can generate 360 unique possibilities.
Name Stephanie can generate 20160 unique possibilities.
Name Ben Nugent can generate 20160 unique possibilities.
